src/backend/catalog/system_views.sql

```diff
@@ -871,6 +871,21 @@ CREATE VIEW pg_stat_activity AS
         LEFT JOIN pg_database AS D ON (S.datid = D.oid)
         LEFT JOIN pg_authid AS U ON (S.usesysid = U.oid);
 
+CREATE VIEW pg_stat_session AS
+    SELECT
+            s.pid,
+            s.total_running_time,
```

Comment:
the question is if we should keep the `total_` prefix. I am actually for dropping it completly. Other opinions ?